sdcm/nemesis.py

```diff
@@ -2658,6 +2718,14 @@ def disrupt(self):
         self.call_random_disrupt_method(disrupt_methods=self.disrupt_methods_list)
 
 
+class ClusterDownScale(Nemesis):
```

Comment:
also, let tried to keep the convention of naming 
`RemovenodeMonkey`

Reply:
Let's change to TerminateAndRemoveNodeMonkey

Reply:
Done